app/models/accounting/transaction.rb

```diff
@@ -80,10 +80,22 @@ class Accounting::Transaction < ApplicationRecord
 
   scope :standard_order, -> {
     joins("LEFT OUTER JOIN options ON options.option_set_id = #{loan_transaction_type_option_set.id}
-      AND options.value = accounting_transactions.loan_transaction_type_value").
-    order(:txn_date, "options.position", :created_at)
+      AND options.value = accounting_transactions.loan_transaction_type_value")
+      .order(:txn_date, "options.position", :created_at)
   }
   scope :interest_type, -> { where(loan_transaction_type_value: LOAN_INTEREST_TYPE) }
```

Comment:
style issues here will be fixed when 10013 and 10014 are merged and this branch is rebased